lib/embroider/update-manifest-from-html.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+const fs = require('fs');
+const findScriptSrcs = require('find-scripts-srcs-in-document');
+
+function getFileName(filePath) {
+  const scriptFileNameRegEx = /([a-zA-Z0-9_\.\-\(\):])+(\.js)/ig;
+  return Array.isArray(filePath.match(scriptFileNameRegEx)) ? filePath.match(scriptFileNameRegEx)[0] : '';
+}
+
+module.exports = function updateManifestFromHtml(root) {
+  // package.json is not present in root location skip
+  // below flow as there is no package.json to update.
+  if (!fs.existsSync(root + '/package.json')) {
```

Comment:
In what scenario, where ember-cli-fastboot is installed as an add-on, will `package.json` be missing?

Reply:
THis was to cover custom app build scenarios where assets are generated in a nested directory in outputPath.

Reply:
I believe this then should cause an error, rather than silently failing for such apps.

Reply:
I removed this check for now. As we discussed, for multiple apps usecase, we can provide a broccoli plugin for updating index.html. I would like to cover that as separate task.